news/337.bugfix

```diff
@@ -0,0 +1,2 @@
+- Fix missing folder_contents icons in classic UI after migration from Plone 5 - see also plone/Prodcuts.CMFPlone#3838
+[szakitibi]
```

Comment:
```suggestion
- Fix missing folder_contents icons in classic UI after migration from Plone 5 - see also plone/Prodcuts.CMFPlone#3838 [szakitibi]
```

It should be in a single line, otherwise it would break on pypi